frontend/ui/network/manager.lua

```diff
@@ -284,26 +289,21 @@ function NetworkMgr:toggleWifiOn(complete_callback, long_press, interactive)
     UIManager:show(toggle_im)
     UIManager:forceRePaint()
 
-    self.wifi_was_on = true
-    G_reader_settings:makeTrue("wifi_was_on")
```

Comment:
That was redundant, the connectivity check will handle it on success, and the connectivity check is *always* run now.

(IIRC, that snippet may predate that fact, hence the duplication).